server/public/model/manifest.go

```diff
@@ -89,6 +89,10 @@ type PluginSetting struct {
 	// and the opposite environment is running the plugin, the setting will be hidden in the admin console UI.
 	// Note that this functionality is entirely client-side, so the plugin needs to handle the case of invalid submissions.
 	Hosting string `json:"hosting"`
+
+	// If true, the setting is sanitized before showing it in the System Console or returning it via the API.
+	// This is useful for settings that contain sensitive information.
+	Secret bool `json:"secret"`
```

Comment:
Slightly unrelated, but since I also recently made some changes to the manifest, I sent an update on the marketplace side -> https://github.com/mattermost/mattermost-marketplace/pull/453 

Just a FYI.